openhtf/util/units/bin/units_from_xls.py

```diff
@@ -95,21 +99,23 @@ def code_generator(sheet):
     
     # Find the indices for the columns we care about.
     for idx, cell in enumerate(rows.next()):
-      if cell.value in COLUMN_NAMES:
+      if cell.value in column_names:
         col_indices[cell.value] = idx
 
     # loop over all remaining rows and pull out units.
     for row in rows:
-      name = row[col_indices[COLUMN_NAMES[0]]].value
-      code = row[col_indices[COLUMN_NAMES[1]]].value
-      suffix = multi_replace(row[col_indices[COLUMN_NAMES[2]]].value,
-                             {"'": "\\\'", '"': '\\\"'})
+      name = row[col_indices[column_names[0]]].value
+      code = row[col_indices[column_names[1]]].value
+      suffix = row[col_indices[column_names[2]]].value.replace("'", "\\\'")
```

Comment:
I believe a least one of `'` or `"` needs to be escaped, since both appear by themselves as suffixes. With triple quoting alone, you end up with either `'''''''` or `"""""""`, which parse as an empty string followed by `'` or `"` respectively.